src/pages/community-champions.js

```diff
@@ -193,6 +199,75 @@ const LeadByExampleTitle = styled(H4)`
     }
 `;
 
+const BenefitsAndRewardsTitle = styled(Title)`
+    margin-top: 0;
+`;
+
+const BenefitsAndRewardsGrid = styled('div')`
+    display: grid;
+    grid-row-gap: ${size.large};
+    @media ${screenSize.mediumAndUp} {
+        grid-column-gap: ${size.mediumLarge};
+        grid-template-columns: repeat(2, 1fr);
+    }
+    @media ${screenSize.largeAndUp} {
+        grid-template-columns: repeat(4, 1fr);
+    }
+`;
+
+const BenefitsAndRewardsItemContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+`;
+
+const BenefitsAndRewardsSectionTitle = styled(H5)`
+    margin-bottom: ${size.default};
+    margin-top: ${size.mediumLarge};
+    text-align: center;
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const BenefitsAndRewardsImageContainer = styled('div')`
+    display: flex;
+    height: ${BENEFITS_AND_REWARDS_IMAGE_CONTAINER_HEIGHT};
+    justify-content: center;
+    max-width: 100%;
+`;
+
+const BenefitsAndRewardsContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.devBlack};
+    margin: 0 -${size.xxlarge};
+    padding: ${size.xlarge} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: 0 -${size.default};
```

Comment:
is this an appropriate use of negative margins? I'm using them so that the black background can extend across the whole page horizontally. the `ContentContainer` which wraps around all of the sections below the hero banner has margins set.

Reply:
I think this is likely the best way for us to go, because alternatives would remove the nice structure we have for the entire page